apps/sms/tools/update_nga_libs.sh

```diff
@@ -12,7 +12,7 @@ bower_path="bower_components"
 
```

Comment:
>  I think it's not that difficult to do it in our build script (or even use npm to setup the lib like threads did. 

Right, I was even thinking about our own fork :) Will check with Wilson if he still plans to rename lib to "bridge", otherwise we'll have to do all that in-house machinery that I'd like to avoid.